### DISCUSSION TITLE
Configuration file added but no meters configured

### DISCUSSION THREAD

**Discussion by catdogmaus** [wmbusmeters/wmbusmeters#758] [Q&A] [answered]
I'm totally confused. I create a Home Assistant wmbusmeters.d/MyTapWater file but wmbusmeters still tells me no meters configured

_Registering meters ...
Generating MQTT configuration ... 
Broker core-mosquitto will be used.
MQTT Discovery cleanup...
Running wmbusmeters ...
Started auto rtlwmbus[00000001] listening on t1
No meters configured. Printing id:s of all telegrams heard!_

what do I miss here
Well I discovered that it is even crazier, whenever I restart wmbusmeters the MyTapWater file is gone.

**Comment by BIBOLV** [comment]
Looks like you are using wmbusmeters as HA addon, so please check [this](https://github.com/weetmuts/wmbusmeters/issues/759) where I tried to explain how to add meter config.

**Reply by catdogmaus** [reply]
Hi
I already got it to work. For me, it seems that documentation about HA integration is wrong or at least confusing:
_Finally, you need to tell Home Assistant how to extract the readings from the MQTT. You can add the following sensor definition into your sensor: section of configuration.yaml._

Create something in configuration.yaml for me means opening that file in file editor and making needed changes. Also it does not help that HA documentation says _see project Readme in github_, and Readme in github says: _Then add a meter file in /etc/wmbusmeters.d/MyTapWater_ I tried to create yaml files but failed, and discovered that addon overwrites by restarting everything so the reason why wmbusmeters told me _no meters configured_ was there was no proper config file. It simply did delete it immediately with every restart.
So I did finaly open addon GUI configuration in yaml, created all setup entries there and everything works now.
It would totally help when somebody would update HA instructions to clear step by step one
Thanks

**Reply by BIBOLV** [reply]
Config interface has changed and docs updated. Please mark as answered in you do not notice any other issues,

**Comment by catdogmaus** [comment, marked as answer]
wmbusmeters has total makeover of all setup options so this thread is not relevant anymore
